k_ext: 5.619899431699218
k_flex: 4.019886875846046
1:
- -0.09496640144692732
- 0.013642970913672827
- 0.03699170632845356
- -0.04812953953455901
- 0.056751893553290286
- 0.1891137734049664
- 0.026908814817800428
- -0.04491234095753082
2:
- 0.2822489151459709
- 0.0473099483755333
- -0.0012398928691920155
- 0.16844027208727821
- 0.007921461394221138
- -0.1944112275085378
- 0.01870409952923712
- 0.1506093421414669
